s3auth-relay/src/main/java/com/s3auth/relay/FtpThread.java

```diff
@@ -84,10 +82,9 @@
     }
 
     /**
-     * Dispatch one request from the encapsulated queue.
-     * @return Amount of bytes sent to socket
-     * @throws InterruptedException If interrupted while waiting for the queue
+     * {@inheritDoc}
      */
+    @Override
     public long dispatch() throws InterruptedException {
         assert this.hosts != null;
```

Comment:
@krzyk These asserts will be romoved when the method will be implemented. They help to prevent warnings related to unused private fields in the class.